src/FamilyHubs.ServiceDirectoryApi.Api/Commands/UpdateOpenReferralService/UpdateOpenReferralServiceCommand.cs

```diff
@@ -467,14 +467,14 @@ private void UpdateTaxonomies(ICollection<OpenReferralService_Taxonomy> existing
             }
             else
             {
-                if (current.Taxonomy != null && updatedServiceTaxonomy.Taxonomy != null)
-                {
-                    current.Taxonomy.Name = updatedServiceTaxonomy.Taxonomy.Name;
-                    current.Taxonomy.Vocabulary = updatedServiceTaxonomy.Taxonomy.Vocabulary;
-                    current.Taxonomy.Parent = updatedServiceTaxonomy.Taxonomy.Parent;
-                }
-
-                currentIds.Add(current.Id);
+            //    if (current.Taxonomy != null && updatedServiceTaxonomy.Taxonomy != null)
```

Comment:
Please remove commented out code if it not needed

Reply:
Removed unused code.